MonicaHelper.py

```diff
@@ -544,7 +544,11 @@ def __init__(self, monica: Monica, firstName: str, lastName: str = None, nickNam
                  deceasedDay: int = None, deceasedMonth: int = None,
                  deceasedYear: int = None, deceasedAgeBased: bool = None,
                  createReminders: bool = True) -> None:
-        genderId = monica.getGenderMapping()[genderType]
+        if genderType == None:
+            genderId = None
+        else:
+            genderId = monica.getGenderMapping()[genderType]
+
```

Comment:
I think this should also work and it's a bit shorter 😃 
Can you confirm this?
```suggestion
        genderId = monica.getGenderMapping().get(genderType, None)
```